src/CSSMotion.tsx

```diff
@@ -142,13 +142,11 @@ export function genCSSMotion(config: CSSMotionConfig) {
 
     function getDomElement() {
       try {
-        // Here we're avoiding call for findDOMNode since it's deprecated
-        // in strict mode. We're calling it only when node ref is not
-        // an instance of DOM HTMLElement. Otherwise use
-        // findDOMNode as a final resort
         return nodeRef.current instanceof HTMLElement
           ? nodeRef.current
-          : findDOMNode<HTMLElement>(wrapperNodeRef.current);
+          : (getDOM(
+              (wrapperNodeRef?.current as any)?.nativeElement,
```

Comment:
`Wrapper` is the component to use `findDOMNode` method. So when we no longer use `findDOMNode` the wrapper can be removed. and just use `getDOM(nodeRef.current)`  instead